src/mscorlib/src/System/Diagnostics/Eventing/EventSource.cs

```diff
@@ -5804,6 +5828,74 @@ public enum EventManifestOptions
         AllowEventSourceOverride = 0x8,
     }
 
+#if FEATURE_EVENTSOURCE_XPLAT 
+
+    internal  class XplatEventLogger : EventListener
```

Comment:
I see.  My thought is that xplat to me feels like a point-in-time statement, because we're starting on Windows and then going x-plat.  Thinking about it as a platform agnostic component vs. a point-in-time thing seems more reasonable to me.  Fine with me to leave as is.